### ISSUE TITLE
Fix `install n with npm` task issue

### ISSUE THREAD

**Issue by chusiang** [chusiang/gitbook.ansible.role#3]
This playbook have this on Ubuntu 14.04 of docker image.

```
TASK [clean npm cache] *********************************************************
task path: /tmp/tasks/setup_node.yml:9
Using module file /usr/lib/python2.7/dist-packages/ansible/modules/commands/command.py
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~ && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440 `" && echo ansible-tmp-1495791427.15-145256880090440="` echo /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440 `" ) && sleep 0'
<localhost> PUT /tmp/tmpvRN13Q TO /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440/command.py
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440/ /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440/command.py && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440/command.py; rm -rf "/root/.ansible/tmp/ansible-tmp-1495791427.15-145256880090440/" > /dev/null 2>&1 && sleep 0'
changed: [localhost] => {
    "changed": true,
    "cmd": [
        "npm",
        "cache",
        "clean"
    ],
    "delta": "0:00:00.695478",
    "end": "2017-05-26 09:37:07.936077",
    "invocation": {
        "module_args": {
            "_raw_params": "npm cache clean",
            "_uses_shell": false,
            "chdir": null,
            "creates": null,
            "executable": null,
            "removes": null,
            "warn": true
        }
    },
    "rc": 0,
    "start": "2017-05-26 09:37:07.240599",
    "stderr": "npm WARN npm npm does not support Node.js v0.10.25\nnpm WARN npm You should probably upgrade to a newer version of node as we\nnpm WARN npm can't make any promises that npm will work with this version.\nnpm WARN npm You can find the latest version at https://nodejs.org/",
    "stderr_lines": [
        "npm WARN npm npm does not support Node.js v0.10.25",
        "npm WARN npm You should probably upgrade to a newer version of node as we",
        "npm WARN npm can't make any promises that npm will work with this version.",
        "npm WARN npm You can find the latest version at https://nodejs.org/"
    ],
    "stdout": "",
    "stdout_lines": []
}

TASK [install n with npm] ******************************************************
task path: /tmp/tasks/setup_node.yml:15
Using module file /usr/lib/python2.7/dist-packages/ansible/modules/packaging/language/npm.py
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~ && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506 `" && echo ansible-tmp-1495791427.97-229401905596506="` echo /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506 `" ) && sleep 0'
<localhost> PUT /tmp/tmpmKHY78 TO /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506/npm.py
<localhost> EXEC /bin/sh -c 'chmod u+x /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506/ /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506/npm.py && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python /root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506/npm.py; rm -rf "/root/.ansible/tmp/ansible-tmp-1495791427.97-229401905596506/" > /dev/null 2>&1 && sleep 0'

```

**Comment by chusiang** [comment]
看來 `node.js`、`npm`、還有 `n` 有改了些東西，暫時不修了。

- https://github.com/mklement0/n-install
- https://github.com/tj/n

**Comment by chusiang** [comment]
Maybe we can install node.js packages with `yarn`, not `npm`.

- https://blog.wu-boy.com/2017/06/downsize-node_modules-to-improve-deploy-speed/

**Comment by chusiang** [comment]
I have fix this issue at Alpine Linux 3.6, but the Ubuntu 14.04 has some problem. :joy:
